client/main.jsx

```diff
@@ -82,6 +82,7 @@ export default class Main extends React.Component {
         <div className="container-fluid" style={{backgroundColor: 'red'}}>
           {widgets}
         </div>
+        <Bart location={this.state.locationTrue}/>
```

Comment:
This is not where widgets go. I'll show you how to check if your component renders